packages/vm/src/instance.rs

```diff
@@ -174,9 +174,17 @@ where
         (get_memory_info(self.wasmer_instance.context()).size as u64) * WASM_PAGE_SIZE
     }
 
-    /// Returns the currently remaining gas
+    /// Returns the currently remaining gas.
+    pub fn get_gas_left(&self) -> u64 {
+        get_gas_left(&self.wasmer_instance)
+    }
+
+    /// Returns the currently remaining gas.
+    ///
+    /// DEPRECATED: renamed to `get_gas_left`.
+    #[deprecated(note = "renamed to `get_gas_left`")]
```

Comment:
👍 Can you add a CHANGELOG entry for this change under a new 0.8.1 section?

Reply:
done in 79a844c